src/components/Projects/TrendingProjectCard.tsx

```diff
@@ -109,26 +111,32 @@ export default function TrendingProjectCard({
           : `/p/${project?.handle}`
       }
     >
-      {metadata ? (
-        <div style={cardStyle} className="clickable-border">
-          <div
-            style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}
-          >
-            <div style={rankStyle}>{rank}</div>
+      <div style={cardStyle} className="clickable-border">
+        <div style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}>
+          <div style={rankStyle}>{rank}</div>
+          {metadata ? (
             <ProjectLogo
               uri={metadata.logoUri}
               name={metadata.name}
```

Comment:
**question**: if we used `?.` here, would we achieve the same outcome as the turnary? 

```suggestion
              uri={metadata?.logoUri}
              name={metadata?.name}
```